lib/Runtime/Base/jitprofiling.cpp

```diff
@@ -6,6 +6,12 @@
 
 #ifdef VTUNE_PROFILING
 
+// For Linux getenv is been defined as PAL_getenv in pal/inc/pal.h
+// We need undef here so that VTune can use it to get environment variable 
+#ifdef PLATFORM_UNIX
+#undef getenv    
```

Comment:
Well, instead of disabling `PAL_getenv` here, we may fix it.

We could replace;
https://github.com/Microsoft/ChakraCore/blob/master/pal/src/cruntime/misc.cpp#L381

to

```
if (pRet == NULL)
    {
        return getenv(name);
    }
    return pRet;
```